tests/components/technove/test_coordinator.py

```diff
@@ -0,0 +1,49 @@
+"""Tests for the coordinator of the TechnoVE integration."""
+
+from datetime import timedelta
+from unittest.mock import MagicMock
+
+from freezegun.api import FrozenDateTimeFactory
+from technove import TechnoVEConnectionError, TechnoVEError
+
+from homeassistant.config_entries import ConfigEntryState
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.update_coordinator import UpdateFailed
+
+from tests.common import MockConfigEntry, async_fire_time_changed
+
+
+async def test_coordinator_connection_error(
+    hass: HomeAssistant,
+    mock_technove: MagicMock,
+    mock_config_entry: MockConfigEntry,
+) -> None:
+    """Test coordinator update connection error."""
+    mock_technove.update.side_effect = TechnoVEConnectionError
+    mock_config_entry.add_to_hass(hass)
+    await hass.config_entries.async_setup(mock_config_entry.entry_id)
+    await hass.async_block_till_done()
+    assert mock_config_entry.state is ConfigEntryState.SETUP_RETRY
```

Comment:
This can just move to the test_init.py file as we are testing init

Reply:
Done.